examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>We can use plot_field for this I think.</p><p>https://github.com/devitocodes/devito/blob/011262a17ceaed9cc34bab8e9667ddb0a0d2d85f/examples/cfd/tools.py#L8</p><p>Also cmap is better to be <code>seismic</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
I know that there are functions to plot the values of receivers and displacament, but these functions using the model structure and in these notebooks we don't use these structure. So I think that is better to use my plot functions and use the seismic cmap.